airbyte-commons-worker/src/main/java/io/airbyte/workers/RecordSchemaValidator.java

```diff
@@ -43,6 +57,16 @@ public void validateSchema(final AirbyteRecordMessage message, final AirbyteStre
     final JsonNode messageData = message.getData();
     final JsonNode matchingSchema = streams.get(messageStream);
 
+    if (workspaceId != null) {
+      if (featureFlagClient.enabled(PerfBackgroundJsonValidation.INSTANCE, new Workspace(workspaceId))) {
+        log.info("feature flag enabled for workspace {}", workspaceId);
```

Comment:
what about the separate threading changes? Or does this PR only include the feature flag changes?

Reply:
also to confirm, we'll opt in workspaces via adding the workspace id in LauchDarkly?

Reply:
This only includes the feature-flag changes.  A quick, follow-up PR will add in the actual changes that will be contained within the feature-flag. 

This allows me to vet that the client is working as expected and seeing updated from LaunchDarkly without having an impact on any users.